schema.sql

```diff
@@ -0,0 +1,381 @@
+-- MySQL Script generated by MySQL Workbench
```

Comment:
I guess this file is not needed for this pull request - if we were to add such files, it would rather be under the sql directory

Reply:
Thanks, I will remove this file as it is no longer required.  But yes, now I see the "mysql" directory.